storey/table.py

```diff
@@ -44,6 +46,20 @@ def __init__(self, table_path: str, storage: Driver, partitioned_by_key: bool =
     def __str__(self):
         return f'{self._container}/{self._table_path}'
 
+    def _get_lock(self, key):
+        key = get_hashed_key(key)
+        cache_element = self._attrs_cache.get(key)
+        if cache_element is None:
+            cache_element = _CacheElement({}, None)
+            self._attrs_cache[key] = cache_element
+        cache_element.lock = Lock()
```

Comment:
this should be in _CacheElement ctor

Reply:
This gets created outside the storey flow (e.g. via `table['abc'] = ...` and in tests), so an asyncio lock can't be created there.

Reply:
then you should move it inside the if so it won't be created each time